### ISSUE TITLE
2.3.0 breaks Jest tests

### ISSUE THREAD

**Issue by RamonGebben** [react-ga/react-ga#163]
After the latest release, `2.3.0` I'm getting this error:
```
<my-path>/node_modules/react-ga/src/index.js:12
    import format from './utils/format';
    ^^^^^^
    SyntaxError: Unexpected token import

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/ScriptTransformer.js:289:17)
```

**Comment by SimeonC** [comment]
Same issue as #162. Closing.

**Comment by RamonGebben** [comment]
Fixed in version 2.3.1
